.github/workflows/pull-request-checks.yaml

```diff
@@ -20,18 +24,19 @@ jobs:
                 ini-values: "memory_limit=-1"
                 php-version: "${{ matrix.php }}"
 
-            - name: Composer install
-              run: composer update --prefer-dist --no-progress --no-suggest --no-interaction --prefer-lowest
+            # Install project
+            - name: Install dependencies
+              run: composer install --prefer-dist --no-progress --no-interaction
 
-            - name: PHPCS Fixer install
-              run: composer bin php-cs-fixer update
+            - name: Composer update
+              run: composer update ${{ matrix.composer-options }}
 
             # Linters
             - name: Run Composer linter
               run: composer validate --strict --no-check-lock
 
-            - name: Run lint PHP
-              run: vendor-bin/php-cs-fixer/vendor/friendsofphp/php-cs-fixer/php-cs-fixer fix --ansi --verbose --diff --dry-run
+            #- name: Run lint PHP
+            #  run: composer run-script php-lint
```

Comment:
Staat deze bewust nog uit?

Reply:
Jep, wil dat in los PR weer aanzetten en daarin het command keertje draaien.

Reply:
Nu toch meegenomen in dit PR.